docs/development/vsphere-dev.md

```diff
@@ -8,6 +8,53 @@ We are using [#sig-onprem channel](https://kubernetes.slack.com/messages/sig-onp
 ## Process
 Here is a [list of requirements and tasks](https://docs.google.com/document/d/10L7I98GuW7o7QuX_1QTouxC0t0aEO_68uHKNc7o4fXY/edit#heading=h.6wyer21z75n9 "Kops-vSphere specification") that we are working on. Once the basic infrastructure for vSphere is ready, we will move these tasks to issues.
 
+## CoreDNS
+Before the support of CoreDNS is stable, use env parameter "VSPHERE_DNS=coredns" to enable using CoreDNS. Or else AWS Route53 will be the default DNS service. To use Route53, follow instructions on: https://github.com/vmware/kops/blob/vsphere-develop/docs/aws.md
+
+For now we hardcoded DNS zone to skydns.local. So your cluster name should have suffix skydns.local.
```

Comment:
When you say 'suffix', do you mean the 'domain name'? 

Will the cluster name looks something like- mycluster.skydns.local?